src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1225,6 +1225,8 @@ type getPeerTelemetryDataMsg struct {
 }
 
 func (m *getPeerTelemetryDataMsg) run(app *app) (interface{}, error) {
+	ctx, _ := context.WithTimeout(app.Ctx, 400*time.Millisecond)
```

Comment:
Probably this is long enough but just noting on other streams the timeout is 30 seconds (maybe it should be lower like this on those too though).

Reply:
I can definitely bump this if we don't think 400ms is enough. I was figuring that we expect these to respond quickly and we don't want to keep these connections open as long.

Reply:
I think it's fine